packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -75,7 +75,7 @@ const PersonalDetails = ({
         onCancel(current_step, goToPreviousStep);
     };
 
-    const is_qualified_for_idv = shouldShowIdentityInformation({
+    const is_for_idv = shouldShowIdentityInformation({
```

Comment:
`is_for_idv` is not descriptive at all, can we rename this?

Reply:
will change the name and leave a comment for clarification.